gallery/src/components/NxThreatCounter/NxThreatCounterPage.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxThreatCounterExample from './NxThreatCounterExample';
+import NxThreatCounterColumnExample from './NxThreatCounterColumnExample';
+import NxThreatCounterGridExample from './NxThreatCounterGridExample';
+
+const NxThreatCounterCode = require('./NxThreatCounterExample?raw'),
+    NxThreatCounterColumnCode = require('./NxThreatCounterColumnExample?raw'),
+    NxThreatCounterGridCode = require('./NxThreatCounterGridExample?raw');
+
+const NxThreatCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small counter which displays a number representing a count of violations, and a short pre-set
+        text string which displays the threat severity.
```

Comment:
This text could use some updating.  It's now a component not a style, and it represents several numbers together not just one.

Reply:
> This text could use some updating. It's now a component not a style, and it represents several numbers together not just one.

https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/e05d98d23ec6c78941170469b2d5eac8cda9fc20